### PR TITLE
[Docs] Add Chinese version of conventions

### PR DESCRIPTION
**Pull request by wangbo-zhao**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

This PR is for chinese version of conventions.md

### CODE REVIEW

File: docs_zh-CN/conventions.md [hunk 1]
```diff
@@ -1 +1,60 @@
 # 默认约定
+如果你想把MMDetection作为自己的项目进行修改的话，请检查下面的约定。
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:2]: 1. In our document, there should be a space between English and Chinese, such as: `把 MMdetection 作为`.
2. 如果你想把 MMDetection 修改为自己的项目，请遵守以下约定。

File: docs_zh-CN/conventions.md [hunk 2]
```diff
@@ -1 +1,60 @@
...
+
+## 损失
+在MMDetection中，`dict` 包含着所有的损失和评价指标，他们将会由`model(**data)`返回。
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:5]: 1. space
2. 在 MMDetection 中，`model(**data)` 的返回值是一个字典，包含着所有的损失及其评价指标。

File: docs_zh-CN/conventions.md [hunk 3]
```diff
@@ -1 +1,60 @@
...
+
+例如，在bbox head中，
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:7]: space

File: docs_zh-CN/conventions.md [hunk 4]
```diff
@@ -1 +1,60 @@
...
+```python
+class BBoxHead(nn.Module):
+    ...
+    def loss(self, ...):
+        losses = dict()
+        # classification loss
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:13]: It is better to translate it, too.

File: docs_zh-CN/conventions.md [hunk 5]
```diff
@@ -1 +1,60 @@
...
+        losses['loss_cls'] = self.loss_cls(...)
+        # classification accuracy
+        losses['acc'] = accuracy(...)
+        # bbox regression loss
+        losses['loss_bbox'] = self.loss_bbox(...)
+        return losses
+```
+
+'bbox_head.loss()'在模型正向阶段会被调用。返回的字典中包含了`'loss_bbox'`, `'loss_cls'`, `'acc'`。只有`'loss_bbox'`, `'loss_cls'`会被用于反向传播，`'acc'`只会被作为评价指标来监控训练过程。
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:22]: 1. 'bbox_head.loss()' 在模型 forward 阶段会被调用
 2. space

File: docs_zh-CN/conventions.md [hunk 6]
```diff
@@ -1 +1,60 @@
...
+
+我们默认，只有那些键的名称中包含'loss'的值会被用于反向传播。这个行为可以通过修改`BaseDetector.train_step()`来改变。
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:24]: space

File: docs_zh-CN/conventions.md [hunk 7]
```diff
@@ -1 +1,60 @@
...
+
+## 空proposals
+在MMDetection中，我们为两阶段mooing中空proposals的情况增加了特殊处理和单元测试。我们同时需要处理整个batch和单一图片中空proposals的情况。例如，在CascadeRoIHead中，
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:27]: 1. there is a typo `mooing`
2. space

File: docs_zh-CN/conventions.md [hunk 8]
```diff
@@ -1 +1,60 @@
...
+```python
+# simple_test method
+...
+# 在整个batch中都没有proposals
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:31]: space

File: docs_zh-CN/conventions.md [hunk 9]
```diff
@@ -1 +1,60 @@
...
+          for j in range(num_imgs):
+                   # 处理空proposals
+                   if rois[j].shape[0] > 0:
+                       bbox_label = cls_score[j][:, :-1].argmax(dim=1)
+                       refine_roi = self.bbox_head[i].regress_by_class(
+                            rois[j], bbox_label[j], bbox_pred[j], img_metas[j])
+                       refine_roi_list.append(refine_roi)
+```
+如果你有自定义的`RoIHead`, 你可以参考上面的方法来处理空proposals的情况。
+If you have customized `RoIHead`, you can refer to the above method to deal with empty proposals.
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:60]: Please delete this line.

**Comment by AronLin** [comment]
Please re-request review if you fix my comments.

File: docs_zh-CN/conventions.md (hunk 3 above)
Comment (AronLin) [reply docs_zh-CN/conventions.md:9]: In our standards, there should be a space between English (including numbers) and Chinese. 
So it should be `在 bbox head 中`.

File: docs_zh-CN/conventions.md [hunk 10]
```diff
@@ -1 +1,65 @@
...
+        # 分类损失
+        losses['loss_cls'] = self.loss_cls(...)
+        # 分类准确率
+        losses['acc'] = accuracy(...)
+        # 边界框损失
+        losses['loss_bbox'] = self.loss_bbox(...)
+        return losses
+```
+
+'bbox_head.loss()'在模型forward阶段会被调用。返回的字典中包含了`'loss_bbox'`, `'loss_cls'`, `'acc'`。只有`'loss_bbox'`, `'loss_cls'`会被用于反向传播，`'acc'`只会被作为评价指标来监控训练过程。
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:25]: Space between English and Chinese.

File: docs_zh-CN/conventions.md (hunk 6 above)
Comment (AronLin) [reply docs_zh-CN/conventions.md:27]: Space between English and Chinese.

File: docs_zh-CN/conventions.md [hunk 11]
```diff
@@ -1 +1,65 @@
...
+
+我们默认，只有那些键的名称中包含'loss'的值会被用于反向传播。这个行为可以通过修改`BaseDetector.train_step()`来改变。
+
+## 空proposals
+
+在MMDetection中，我们为两阶段方法中空proposals的情况增加了特殊处理和单元测试。我们同时需要处理整个batch和单一图片中空proposals的情况。例如，在CascadeRoIHead中，
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:31]: Space between English and Chinese.

File: docs_zh-CN/conventions.md (hunk 8 above)
Comment (AronLin) [reply docs_zh-CN/conventions.md:37]: Space between English and Chinese.

File: docs_zh-CN/conventions.md [hunk 12]
```diff
@@ -1 +1,65 @@
...
+        mask_classes = self.mask_head[-1].num_classes
+        segm_results = [[[] for _ in range(mask_classes)]
+                        for _ in range(num_imgs)]
+        results = list(zip(bbox_results, segm_results))
+    else:
+        results = bbox_results
+    return results
+...
+
+# 在单张图片中没有proposals
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:53]: Space between English and Chinese.

File: docs_zh-CN/conventions.md [hunk 13]
```diff
@@ -1 +1,65 @@
...
+for i in range(self.num_stages):
+    ...
+    if i < self.num_stages - 1:
+          for j in range(num_imgs):
+                   # 处理空proposals
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:58]: Space between English and Chinese.

File: docs_zh-CN/conventions.md [hunk 14]
```diff
@@ -1 +1,65 @@
...
+                   if rois[j].shape[0] > 0:
+                       bbox_label = cls_score[j][:, :-1].argmax(dim=1)
+                       refine_roi = self.bbox_head[i].regress_by_class(
+                            rois[j], bbox_label[j], bbox_pred[j], img_metas[j])
+                       refine_roi_list.append(refine_roi)
+```
+如果你有自定义的`RoIHead`, 你可以参考上面的方法来处理空proposals的情况。
```
Comment (AronLin) [inline docs_zh-CN/conventions.md:65]: Space between English and Chinese.

**Review by AronLin — COMMENTED** [review]
Please add space between English(including numbers) and Chinese. This is a unified standard in MMDetection.

**Review by AronLin — APPROVED** [review]

**Comment by AronLin** [comment]
Kindly ping @BIGWangYuDong for the final check.

File: docs_zh-CN/conventions.md [hunk 15]
```diff
@@ -1 +1,65 @@
...
+        # 分类损失
+        losses['loss_cls'] = self.loss_cls(...)
+        # 分类准确率
+        losses['acc'] = accuracy(...)
+        # 边界框损失
+        losses['loss_bbox'] = self.loss_bbox(...)
+        return losses
+```
+
+'bbox_head.loss()' 在模型 forward 阶段会被调用。返回的字典中包含了 `'loss_bbox'`,`'loss_cls'`,`'acc'`。只有 `'loss_bbox'`, `'loss_cls'` 会被用于反向传播，`'acc'` 只会被作为评价指标来监控训练过程。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/conventions.md:25]: 'bbox_head.loss()' -> `'bbox_head.loss()'` (add \`  before and after the text)

File: docs_zh-CN/conventions.md [hunk 16]
```diff
@@ -1 +1,65 @@
...
+
+我们默认，只有那些键的名称中包含 'loss' 的值会被用于反向传播。这个行为可以通过修改 `BaseDetector.train_step()` 来改变。
```
Comment (BIGWangYuDong) [inline docs_zh-CN/conventions.md:27]: 'loss' -> `'loss'` (add ` before and after the text)

**Comment by BIGWangYuDong** [comment]
LGTM, two details need to be modified
